src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -209,6 +193,21 @@ public Response describeDataSets(@Auth AuthUser authUser) {
         }
     }
 
+    @GET
+    @Produces("application/json")
+    @PermitAll
+    @Path("/v2")
+    public Response findAllDatasetsByUser(@Auth AuthUser authUser) {
```

Comment:
Nit: may be more accurate to name this method `findAllDatasetsAvailableToUser` or something similar since my first assumption when reading this name is that the method would find datasets where the `createUser` matches the provided user. Similar comment for `datasetService.findAllDatasetsByUser(user)`.